packages/experimental/mosaic/src/testing/testing.ts

```diff
@@ -8,7 +8,7 @@ import { Item, Location } from '../layout';
 
 export const createItem = (location?: Location): Item => ({
   id: faker.datatype.uuid(),
-  label: faker.lorem.words(3),
+  title: faker.lorem.words(3),
   content: faker.lorem.sentences(faker.datatype.number(3)),
```

Comment:
On the above topic, I’d reconsider `content` since it’s possibly too vague. Maybe `body` is a good candidate.

Reply:
However this is a test class -- so it's representative of some end user concept not the framework. so on that basis it would be better to feel like a real world data type.